### PR TITLE
Added LinkedList Topic in DSA

### PR DESCRIPTION
**Pull request by SadafKausar2025**
## Related Issue #950

[Cite any related issue(s) this pull request addresses. If none, simply state "None”]

## Description

[Please include a brief description of the changes or features added]

## Type of PR

- [x] Documentation update

## Screenshots / Videos (if applicable)

https://github.com/CodeHarborHub/codeharborhub.github.io/assets/137484958/564d8a2d-b060-4ec3-82ce-376d5a96bb78

Screenshot 2024-06-10 221248

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
now update code based on suggestions

File: docs/dsa/linkedlist/LinkedList.md [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+
+Linked lists are important because they provide a flexible way to store and manipulate data. They allow for efficient insertion and deletion of elements, which can be particularly useful in applications where the size of the data set changes frequently.
+
+## How to declare a Linked List?
+
+A linked list can be declared in various programming languages using the following syntax:
+
+# LinkedList in Data Structures and Algorithms (DSA)
+
+## Table of Contents
```
[the remark this answers is not in the document]
Comment (SadafKausar2025) [reply docs/dsa/linkedlist/LinkedList.md:51]: ok

**Comment by SadafKausar2025** [comment]
hello @Ajay-Dhangar 
done with the changes
can you please review this PR?

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**